src/safeposix/syscalls/sys_calls.rs

```diff
@@ -53,7 +54,9 @@ impl Cage {
         }
         let cageobj = Cage {
             cageid: child_cageid, cwd: interface::RustLock::new(self.cwd.read().unwrap().clone()), parent: self.cageid,
-            filedescriptortable: interface::RustLock::new(newfdtable)
+            filedescriptortable: interface::RustLock::new(newfdtable),
+            getgid: AtomicI32::new(self.getgid.load(Ordering::Relaxed)), getuid: AtomicI32::new(self.getuid.load(Ordering::Relaxed)), getegid: AtomicI32::new(self.getegid.load(Ordering::Relaxed)), geteuid: AtomicI32::new(self.geteuid.load(Ordering::Relaxed))
```

Comment:
There _should_ be a simpler way to do this. It essentially does `getuid: self.getuid`